migrations/20231011151001-change-org-self-auth-to-boolean.js

```diff
@@ -0,0 +1,13 @@
+const TABLE = 'organization';
+
+exports.up = async db => {
+  await db.addColumn(TABLE, 'isSelfAuthorized', { type: 'boolean', notNull: true, defaultValue: false });
+  await db.runSql('update "organization" set "isSelfAuthorized" = TRUE where "selfAuthorizedAt" is not null');
+  await db.removeColumn(TABLE, 'selfAuthorizedAt');
+};
+
+exports.down = async db => {
+  await db.addColumn(TABLE, 'selfAuthorizedAt', { type: 'timestamp', allowNull: true });
+  await db.runSql('update "organization" set "selfAuthorizedAt" = CURRENT_DATE where "isSelfAuthorizedAt" IS TRUE');
```

Comment:
Need to update the column name to `isSelfAuthorized` not `isSelfAuthorizedAt`

Reply:
Now that's just embarrassing.